README.md

```diff
@@ -47,11 +46,13 @@ const fetchProfile = async (publicKey: string) => {
 #### Login
 
 ```js
-import { identity } from "deso.js";
+import { Identity } from "deso.js";
+
+const identity = new Identity({ referralCode: 'WSZRQFPY' });
```

Comment:
Do you think this is worth implementing a Singleton pattern?

Reply:
Yea think it would make sense to go with a singleton pattern to avoid problems. I can't really think of a good use case where someone might want multiple identity instances registered. I can implement a quick change to switch to that pattern (unless you get to it first)

Reply:
Just merged this and just set up a new PR to move to singleton creation instead: https://github.com/bitclouthunt/deso.js/pull/15